README.md

```diff
@@ -47,8 +52,6 @@ Execute the commands below to build from the source.
 
 As an open source project, Ballerina welcomes contributions from the community. 
 
-You can also check for [open issues](https://github.com/ballerina-platform/module-ballerinax-kafka/issues) that interest you. We look forward to receiving your contributions.
-
```

Comment:
Shall we add the link for the `ballerina-standard-library` repo issues, filtered with the label `module/kafka`?

Reply:
As discussed offline, this is being removed